### PR TITLE
Added missing supplementary material (#264 closes #535)

### LINKED ISSUE #535: Missing supplementary material
Hi all, thanks for this awesome system!

I noticed that the supplementary material of the following EMNLP 2012 paper is missing in ACL Anthology which I remember was there and could be downloaded in some earlier version of ACL Anthology.

Anthology ID: D12-1057
https://www.aclweb.org/anthology/D12-1057/
Excitatory or Inhibitory: A New Semantic Orientation Extracts Contradiction and Causality from the Web

Did all the supplementary stuff move to elsewhere?

### PR DESCRIPTION
**Pull request by mjpost**
This adds missing attachments. I also had to change the schema because there is at least one frontmatter attachment. I haven't done much sanity checking.

### CODE REVIEW

**Comment by mjpost** [comment]
I didn't realize that we had moved away from `<attachment type="X">` to `<X>` (e.g, `X=software`). I will have to update this.

**Comment by mjpost** [comment]
Actually, it's not clear to me which is canonical. @mbollmann, can you direct here? Should I be using

    <software>...

or

    <attachment type="software">...

?

**Comment by mbollmann** [comment]
Both work and do the same thing, since both were used in the XML when I wrote the code. I'm not aware that we've ever established a canonical version.

**Comment by mbollmann** [comment]
That said I'd vote for `attachment` since it doesn't require us to add new XML tags when there's a new attachment type.

**Comment by mjpost** [comment]
Okay, cool, I'll convert them all in this PR.

**Comment by akoehn** [comment]
If you convert all, can you update the schema to disallow the other variant?

**Comment by davidweichiang** [comment]
Agree, `attachment` is better.

**Comment by knmnyn** [comment]
Agreed that `attachment` is better as well.  It'll make for much easier
processing downstream.

**Comment by mjpost** [comment]
Done.

File: data/xml/P14.xml [hunk 1]
```diff
@@ -2633,7 +2654,8 @@
       <pages>706–711</pages>
       <url>P14-2115</url>
       <doi>10.3115/v1/P14-2115</doi>
-      <dataset>P14-2115.Datasets.tgz</dataset>
+      <attachment type="dataset">P14-2115.Datasets.tgz</attachment>
+      <attachment type="datasets">P14-2115.Datasets.tgz</attachment>
```
Comment (mbollmann) [inline data/xml/P14.xml]: Seems that this attachment type is added twice?

File: data/xml/W19.xml [hunk 2]
```diff
@@ -2126,9 +2129,10 @@
...
       <url>W19-2009</url>
-      <software>W19-2009.Software.zip</software>
+      <attachment type="software">W19-2009.Software.zip</attachment>
       <attachment type="poster">W19-2009.Poster.pdf</attachment>
       <doi>10.18653/v1/W19-2009</doi>
+      <attachment type="software">W19-2009.Software.zip</attachment>
```
Comment (mbollmann) [inline data/xml/W19.xml]: Actually, this is too...

File: data/xml/P14.xml (hunk 1 above)
Comment (mjpost) [reply data/xml/P14.xml]: Oh, drat. I checked for missing attachments before the tag conversion.

**Comment by mjpost** [comment]
Anyone have a minute to check this over? It should be pretty non-controversial—I just

- converted <dataset>, <software>, and <presentation> tags to <attachment>
- added attachments that were on the server but were not in the XML
- added in the script I used to do it

I verified by script that there are no duplicate attachments.

**Comment by mjpost** [comment]
Ping @acl-org/acl-style @mbollmann @aryamccarthy @davidweichiang @akoehn

**Pull request merged**